extension/client.go

```diff
@@ -29,6 +30,25 @@ const (
 	extensionIdentifierHeader = "Lambda-Extension-Identifier"
 )
 
+var (
+	// set up logging defaults for our own logging output
+	log = logrus.WithFields(logrus.Fields{
+		"source": "hny-lambda-ext-client",
+	})
+)
+
+func init() {
+	envLogLevel, ok := os.LookupEnv("HNY_LOG_LEVEL")
+	if !ok {
+		envLogLevel = "info"
+	}
+	parsedLogLevel, err := logrus.ParseLevel(envLogLevel)
+	if err != nil {
+		parsedLogLevel = logrus.InfoLevel
+	}
+	logrus.SetLevel(parsedLogLevel)
```

Comment:
Feels weird to me to repeat this initialization in each package. Is there a better way for me to determine the level once, even if the level must be _set_ individually in each place?

Reply:
It seems so! I left the `init()` in main, removed the `init()` function from other packages, and was still getting log output for the level set. Horrah!

Reply:
Updated with just one `init()` to parse and set log level. 🌳 🪓